tests/unit/utils/test_openstack.py

```diff
@@ -41,7 +41,7 @@ def test_version_range_raises_ValueError(lower, upper):
         (
```

Comment:
Please fill up the other parametrization tests for keystone, gnocchi and etc

Reply:
Done